README.md

```diff
@@ -1,52 +1,52 @@
-SCiMMA Client
+Hop
 =============
 
-![](https://github.com/scimma/client_library/workflows/build/badge.svg)
+![](https://github.com/scimma/hop/workflows/build/badge.svg)
 
-**scimma-client** is a pub-sub client library for Multimessenger Astrophysics.
+**hop** is a pub-sub client library for Multimessenger Astrophysics.
```

Comment:
We may want to refer to the package in the user-facing docs as **hop-client** to match the name on the package repos (even though the code is restructured to be `hop/` rather than `hop/client/` -- thoughts?

Reply:
I think that sounds reasonable. I would also propose naming the repo scimma/hop-client to reflect this so it's all consistent.